GAMES_LIST.md

```diff
@@ -462,5 +462,4 @@ EOS does not provide players data.
 
 ### <a name="palworld"></a>Palworld
 Palworld support can be unstable, the devs mention the api is currently experimental.  
-To query palworld servers, the `RESTAPIEnabled` must be `True` in the configuration file, and you need to pass
-the `username` (currently always `admin`) and `password` (settings parameter) for it.
+To query Palworld servers, the `RESTAPIEnabled` setting must be `True` in the configuration file, and you need to pass the `username` (currently always `admin`) and the `adminpassword` (settings parameter) for it.
```

Comment:
So, `adminpassword` its what is in the game's configuration?
I think this needs another rewording to explicitly mention that you need to pass a `password` parameter to gamedig from the `adminpassword` setttings parameter.

Reply:
See https://github.com/jammsen/docker-palworld-dedicated-server/blob/develop/docs/ENV_VARS.md and https://tech.palworldgame.com/settings-and-operation/configuration - The Value is just called adminpassword or ArminPassword. People with administrative access to the gameserver should understand what is asked of them.

Reply:
I understand, but I meant that the wording isn't really great here, yes the value is called `adminpassword` but we need to pass it to gamedig as the `password` field.